providers/json-binding/src/main/java/org/jboss/resteasy/plugins/providers/jsonb/JsonBindingProvider.java

```diff
@@ -37,14 +43,52 @@
 @Priority(Priorities.USER-100)
 public class JsonBindingProvider extends AbstractJsonBindingProvider
         implements MessageBodyReader<Object>, MessageBodyWriter<Object> {
-
+   private static final boolean JSONB_DISABLED = Boolean.getBoolean("resteasy.jsonb.disable");
```

Comment:
`resteasy.jsonb.disable` is not documented

Reply:
This is in my todo list. I'll update the documentation soon.